playbooks/openshift-console/private/redeploy-certificates.yml

```diff
@@ -20,16 +20,5 @@
       state: absent
       namespace: openshift-console
```

Comment:
The install playbook forces a rollout, so we can probably skip the step that deletes the pods.

Reply:
Right, it should not be necessary - deployment would remove the old ones these correctly anyway